tests/fixtures/census_schedule/calendar-listview.html

```
@@ -0,0 +1,42 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)